server/models/User.js

```diff
@@ -198,6 +198,7 @@ export default (Sequelize, DataTypes) => {
             emailWaitingForValidation: this.emailWaitingForValidation,
             createdAt: this.createdAt,
             updatedAt: this.updatedAt,
+            emailConfirmationToken: this.emailConfirmationToken,
```

Comment:
This is a private information that should never be shared, it should not appear in `info`.

Because there's only one email relying on that, I would suggest passing it manually where it's needed.